api/generate_models.ts

```diff
@@ -14,7 +14,8 @@ async function generateApi(
   const api: Spec = await SwaggerParser.bundle(`api/${specFileName}.yaml`);
 
   const specCode = `
-    // tslint:disable:object-literal-sort-keys
+    /* tslint:disable:object-literal-sort-keys */
```

Comment:
this C-style of comments is needed or tslint would complain that there's "commented code"